cursorless-talon/src/actions/actions.py

```diff
@@ -40,22 +40,10 @@ def cursorless_command(action_id: str, target: dict):
         """Perform cursorless command on target"""
         if action_id in callback_action_map:
             return callback_action_map[action_id](target)
-        elif action_id in makeshift_action_map:
-            command, command_options, talon_options = makeshift_action_map[action_id]
-            return_value = (
-                vscode_command(command, target, command_options)
-                if talon_options.await_command
-                else vscode_command_no_wait(command, target, command_options)
-            )
-
-            if talon_options.post_command_sleep_ms:
-                actions.sleep(f"{talon_options.post_command_sleep_ms}ms")
-
-            return return_value
         elif action_id in no_wait_actions:
-            return actions.user.cursorless_single_target_command_no_wait(
-                action_id, target
-            )
+            actions.user.cursorless_single_target_command_no_wait(action_id, target)
```

Comment:
How come you're dropping the return value?

Reply:
Since we don't wait it cannot return a value. In fact the action it's calling does not have a return.